indico/modules/users/client/js/dashboard.jsx

```diff
@@ -0,0 +1,33 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import {Translate} from 'indico/react/i18n';
+
+import ICSCalendarLink from './ICSCalendarLink';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const userId = document.querySelector('body').dataset.userId;
+  ReactDOM.render(
+    <ICSCalendarLink
+      endpoint="users.export_dashboard_ics"
+      urlParams={{user_id: userId}}
+      options={[
+        {key: 'events', text: Translate.string('Events at hand'), queryParams: {linked: 'true'}},
```

Comment:
Do we have to use `'true'`? Wouldn't that work as well if we passed `true`?

Reply:
Query params can only be strings, so I find it less confusing if we pass them as a string, as opposed to relying on whatever implicit mechanism to do so.

Reply:
Yeah, I was just curious whether those query params get converted to string automatically an exception is thrown in case you pass something else than a string.